packages/fxa-settings/src/pages/Signin/InlineRecoverySetup/index.tsx

```diff
@@ -0,0 +1,225 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { logPageViewEvent } from '../../../lib/metrics';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../../models';
+import InputText, { InputTextProps } from '../../../components/InputText';
+import { ReactComponent as CopyIcon } from './recovery_code_copy.svg';
+import { ReactComponent as DownloadIcon } from './recovery_code_download.svg';
+import { ReactComponent as PrintIcon } from './recovery_code_print.svg';
+import { ReactComponent as RecoveryCodesGraphic } from '../SigninRecoveryCode/graphic_recovery_codes.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { Localized } from '@fluent/react';
```

Comment:
Looks like {InputTextProps}, LinkExternal and {Localized} aren't used. Could remove those imports :)